src/main/resources/jmetemplate/[IF=MULTIPLATFORM]/ios-moe[IF=JME_IOSMOE]/src/main/java/[GAME_PACKAGE]/ios/DummyMain.java

```diff
@@ -0,0 +1,78 @@
+package [GAME_PACKAGE];
+
+import org.moe.natj.general.Pointer;
+import org.moe.natj.general.ann.RegisterOnStartup;
+import org.moe.natj.objc.ann.Selector;
+
+import apple.NSObject;
+import apple.foundation.NSDictionary;
+import apple.corefoundation.struct.CGRect;
+import apple.uikit.UIApplication;
+import apple.uikit.UIColor;
+import apple.uikit.UIImage;
+import apple.uikit.UINavigationController;
+import apple.uikit.UIScreen;
+import apple.uikit.UIViewController;
+import apple.uikit.UIWindow;
+import apple.uikit.c.UIKit;
+import apple.uikit.enums.UIBarStyle;
+import apple.uikit.protocol.UIApplicationDelegate;
+
+@RegisterOnStartup
+public class DummyMain extends NSObject implements UIApplicationDelegate
+{
+    public static void main(String[] args) {
+        UIKit.UIApplicationMain(0, null, null, DummyMain.class.getName());
+    }
+
+    @Selector("alloc")
+    public static native DummyMain alloc();
+
+    protected DummyMain(Pointer peer) {
+        super(peer);
+    }
+
+    private UIWindow window;
+
+    @Override
+    public boolean applicationDidFinishLaunchingWithOptions(UIApplication application, NSDictionary launchOptions) {
+        UIScreen screen = UIScreen.mainScreen();
+        CGRect bounds = screen.bounds();
+        window = UIWindow.alloc().initWithFrame(bounds);
+
+        UIViewController vc = MoeIosJmeAppHarness.alloc().init();
+
+        // Initialization with navigation bar
```

Comment:
Is this meant to be commented out?

Reply:
Yes, I left it as an example on how to add a top bar. I think most, if not all, people will use fullscreen without any bar, but just in case